### PR TITLE
Milestone 3: Mobile first (adding contact form) with collaboration

### PR DESCRIPTION
**Pull request by MarwanDev**
## In this milestone we:

-  Added contact form
-  Added a footer that contains social media icons

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -219,5 +219,27 @@ <h2>Skills</h2>
...
       </section> 
+
+      <section class="contact-section">
+        <div class="contact-head">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</div>
+        <div class="form">
+            <input class="form-input" type="text" name="name" placeholder="Full name">
+            <input class="form-input" type="email" name="email" placeholder="Email address">
+            <textarea name="mail-message" id="" cols="20" rows="5" placeholder="Enter text here"></textarea>
+            <input class="send" type="submit" value="Get in touch">
+        </div>
```
Comment (GedeonTS) [inline index.html]: - [ ] Kindly make sure your form is wrapped in a <form> container for semantic and SEO purposes :+1: 
- [ ] Kindly make sure you have form validations as requested in the project requirements [here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#:~:text=Make%20sure%20that%3A,the%20fields%20is%20empty.)
- [ ] As a requirement for this milestone, you have to post data using the [formspree](https://formspree.io/html) services.

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @MarwanDev  and @AnsarIbrahim   , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- No linter errors :heavy_check_mark: 
- Good use of GitHub flow :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (AlphaNtihinduka) [inline index.html]: - [ ] As the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) suggest, kindly ensure that when the submit button is clicked the form inputs are collected. I suggest that you collect data by using the [Formspree service](https://formspree.io/html) as it has been clearly stated ☺️ .

File: index.html [hunk 2]
```diff
@@ -219,5 +219,27 @@ <h2>Skills</h2>
...
       </div>
       </section> 
+
+      <section class="contact-section">
+        <div class="contact-head">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</div>
+        <div class="form">
+            <input class="form-input" type="text" name="name" placeholder="Full name">
+            <input class="form-input" type="email" name="email" placeholder="Email address">
+            <textarea name="mail-message" id="" cols="20" rows="5" placeholder="Enter text here"></textarea>
+            <input class="send" type="submit" value="Get in touch">
```
Comment (AlphaNtihinduka) [inline index.html]: - [ ] Additionally, let's focus on making the form inputs more professional and easily accessible. Again we shall refer to the project requirements. I request to put the attached points into consideration.
- Make sure that:
  - Name field should accept up to 30 chars.
  - Message field accepts up to 500 chars.
  - A user should not be able to submit the form if one or more of the fields is empty.

File: styles.css [hunk 3]
```diff
@@ -387,3 +387,106 @@ hr {
...
+  background-color: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+.contact-section hr {
+  margin: 100px auto 0 auto;
+  width: 100%;
+  background-color: #dfe1e6;
+}
```
Comment (AlphaNtihinduka) [inline styles.css:471]: - [ ] As you can see 😎 in the Figma template design, the `hr` covers 100% width of the page. In addition, the social icons are close to the `hr`. Therefore as a way of sticking to the Figma design including the font, colors, images, and spacing (margin/padding), I kindly request make sure that you change the requested changes to match the Figma design 🥰 .

Your design 😎 
image


Figma design 😎 
image

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @MarwanDev 🙋‍♂️
Good job working on your first contact form. You have done the design to the best expectation and incredibly well. 👏  

However, there are some issues mostly on the input attributes for [semantic HTML](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwjwyP2CrLD8AhUZi_0HHdAIB8UQFnoECAsQAQ&url=https%3A%2F%2Fwww.w3schools.com%2Fhtml%2Fhtml5_semantic_elements.asp&usg=AOvVaw38SC2etR0JeORCKyF1fqbU) practice and font colors. 
You still need to implement few on the project but you are almost there!!! :watch: 

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Optional suggestions
- [ ] [OPTIONAL]  Consider the use of [`<fieldset>`](https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_structure_a_web_form#the_fieldset_and_legend_elements)
If you notice for form [accessibility](https://developer.mozilla.org/en-US/docs/Learn/Accessibility) reasons wrapping the inputs in the `<fieldset>` is more semantic. Kindly make the form more accessible to screen readers such us [Jaws](https://www.freedomscientific.com/products/software/jaws/) and [NVDA](https://www.nvaccess.org) by using `<fieldset>`.



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: index.html [hunk 4]
```diff
@@ -219,5 +219,27 @@ <h2>Skills</h2>
         </ul>
       </div>
       </section> 
+
+      <section class="contact-section">
+        <div class="contact-head">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</div>
+        <div class="form">
+            <input class="form-input" type="text" name="name" placeholder="Full name">
```
Comment (osamaashraf6) [inline index.html]: - [ ] As stated by the previous reviewer kindly As the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) suggest, kindly ensure that when the submit button is clicked the form inputs are collected. I suggest that you collect data by using the [Formspree service](https://formspree.io/html) as it has been clearly stated ☺️.

File: index.html (hunk 2 above)
Comment (osamaashraf6) [inline index.html]: - [ ] As stated 😃 by the previous reviewer kindly Name input (can be first name and last name field depending on the template used) has maxlength="30" attribute.
Message input has maxlength="500" attribute.💯
All input fields have the required attribute to activate the validation on submission.

**Review by osamaashraf6 — CHANGES_REQUESTED** [review]
Hi  @MarwanDev ☺️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

you-are-near

## Highlights ✔️

- [x] Descriptive READme file in a professional way.
- [x] No linter errors. 💯
- [x] Good usage in correcting GitHub flow

 ## Required Changes ♻️
- Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (MarwanDev) [reply index.html]: Does that mean that the name needs to be divided into 2 inputs: first and last name?

**Comment by AnsarIbrahim** [comment]
Yes see the desktop version it will have two input one is first name
another is last name

File: index.html [hunk 5]
```diff
@@ -219,5 +219,28 @@ <h2>Skills</h2>
...
       </div>
       </section> 
+
+      <section class="contact-section">
+        <p class="contact-head">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        <form class="form" action="https://formspree.io/f/mvonaqyb" method="POST">
+            <input class="form-input" type="text" name="name" placeholder="First name" maxlength="30">
+            <input class="form-input" type="text" name="name" placeholder="Last name" maxlength="30">
+            <input class="form-input" type="email" name="email" placeholder="Email address" maxlength="30">
+            <textarea name="mail-message" id="" cols="20" rows="5" placeholder="Enter text here" maxlength="500"></textarea>
```
Comment (Shisui6) [inline index.html]: As stated by the previous reviewer, kindly ensure that the user cannot submit the form if one of the input fields are empty, as it is one of the project requirements🙏:

image

Currently, your form can be submitted without providing any input. You can fix this by adding a `required` attribute to all your input fields

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @MarwanDev and @AnsarIbrahim, 

# Requires Changes 🍀
Great work on making the changes requested by a previous reviewer 👍
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet🪲.


 ## Highlights
 - Good use of flexbox ✅
 - HTML validations done correctly ✅
 - Good design ✅

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by divinecharlotte — APPROVED** [review]
Hi @MarwanDev , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap
## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**